README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
This is one of the reasons why I questioned the difference between `oss-generic` and `nus-oss` even back then.

> ...OSS projects **we** have on GitHub"

"We" here represents the team from NUS-OSS, and in no way we are _the_ group appointed by GitHub to be some kind of ambassador. We may have set up a workflow followed in all our projects, we may welcome all other projects to adopt/adapt (as mentioned soon after), but in no way we are _the_ authority for GitHub's OSS workflow (if it even exists).